Fix io.vertx.ext.hawkular.impl.NamedPoolITest.testNamedPoolMetricsValues

```
java.lang.AssertionError: Expected mars01.host13.vertx.pool.worker.test-worker.delay value 1140 to be greater than 1200
	at org.junit.Assert.fail(Assert.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrap.invoke(StaticMetaMethodSite.java:133)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:102)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:56)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:194)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:206)
	at io.vertx.ext.hawkular.impl.BaseITest.assertCounterGreaterThan(BaseITest.groovy:174)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
	at groovy.lang.MetaClassImpl.invokeStaticMethod(MetaClassImpl.java:1466)
	at org.codehaus.groovy.runtime.callsite.StaticMetaClassSite.callStatic(StaticMetaClassSite.java:65)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:56)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:194)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:222)
	at io.vertx.ext.hawkular.impl.NamedPoolITest.testNamedPoolMetricsValues(NamedPoolITest.groovy:67)
```

--- a/src/test/groovy/io/vertx/ext/hawkular/impl/NamedPoolITest.groovy
+++ b/src/test/groovy/io/vertx/ext/hawkular/impl/NamedPoolITest.groovy
@@ -64,7 +64,10 @@ class NamedPoolITest extends BaseITest {
 
     async.awaitSuccess()
 
-    assertCounterGreaterThan(maxPoolSize * 3 * sleepMillis, tenantId, "${baseNameWithPoolName}delay")
+    // If all tasks could be submitted *exactly* at the same time, cumulated delay would be maxPoolSize * 3 * sleepMillis
+    // In practice, the value will be close, but not equal
+    // So let's make sure it's at least maxPoolSize * 2 * sleepMillis
+    assertCounterGreaterThan(maxPoolSize * 2 * sleepMillis, tenantId, "${baseNameWithPoolName}delay")
     assertGaugeEquals(0, tenantId, "${baseNameWithPoolName}queued")
     assertCounterEquals(taskCount, tenantId, "${baseNameWithPoolName}queuedCount")
     assertCounterGreaterThan(taskCount * sleepMillis, tenantId, "${baseNameWithPoolName}usage")

